src/controller/python/chip-device-ctrl.py

```diff
@@ -41,7 +41,8 @@
 from chip.ChipBleUtility import FAKE_CONN_OBJ_VALUE
 from chip.setup_payload import SetupPayload
 from xmlrpc.server import SimpleXMLRPCServer
-
+from enum import Enum
```

Comment:
We should avoid deletions as much as possible for this Repo

Reply:
Resolved